BuildResidentialHPXML/measure.rb

```diff
@@ -1382,6 +1382,72 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('W')
     args << arg
 
+    perf_data_capacity_type_choices = OpenStudio::StringVector.new
+    perf_data_capacity_type_choices << 'Capacity'
+    # perf_data_capacity_type_choices << 'CapacityFractionOfNominal'
+
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('perf_data_capacity_type', perf_data_capacity_type_choices, false)
+    arg.setDisplayName('Detailed Performance Data: Capacity Type')
+    arg.setDescription('Capacity type of detailed performance data.')
+    arg.setUnits('Capacity')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heating_perf_data_outdoor_temperatures', false)
+    arg.setDisplayName('Detailed Performance Data: Heating Outdoor Temperatures')
+    arg.setDescription('Outdoor temperatures of heating detailed performance data. If multiple performance data points, use a comma-separated list.')
```

Comment:
"If multiple performance data points, use a comma-separated list."

For all of these arguments, multiple data points are _required_. So perhaps the language should be something like: "At least two performance data points are required using a comma-separated list."